playwright/login.test.ts

```diff
@@ -1,23 +1,11 @@
-jest.setTimeout(60e3);
+import { test } from "@playwright/test";
 
-test("login with pro@example.com", async () => {
-  const context = await browser.newContext();
-  const page = await context.newPage();
-  await page.goto("http://localhost:3000/auth/login");
-  // Click input[name="email"]
-  await page.click('input[name="email"]');
-  // Fill input[name="email"]
-  await page.fill('input[name="email"]', `pro@example.com`);
-  // Press Tab
-  await page.press('input[name="email"]', "Tab");
-  // Fill input[name="password"]
-  await page.fill('input[name="password"]', `pro`);
-  // Press Enter
-  await page.press('input[name="password"]', "Enter");
+// Using logged in state from globalSteup
+test.use({ storageState: "playwright/artifacts/proStorageState.json" });
 
+test("login with pro@example.com", async ({ page }) => {
+  // Try to go homepage
+  await page.goto("http://localhost:3000/");
```

Comment:
Isn't the baseURL configured? Then "/" is enough

Reply:
Totally missed this, thank you!

```suggestion
  await page.goto("/");
```